core/tests/test_utils.py

```diff
@@ -197,6 +199,206 @@ def get_storage_model_classes():
                     yield clazz
 
 
+class ElasticSearchServicesStub(python_utils.OBJECT):
```

Comment:
Do we need a test for the stub class? (considering this is mocking our own code not a thrid party code and we are expected to change it as per the changes made in our own elasetic_seacrh_service file, I think we should have a test for this.) [not sure!]